Fabric.Identity.IntegrationTests/IdentityProvidersControllerTests.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System.Collections.Generic;
+using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Fabric.Identity.API.Models;
+using Newtonsoft.Json;
+using Xunit;
+
+namespace Fabric.Identity.IntegrationTests
+{
+    public class IdentityProvidersControllerTests : IntegrationTestsFixture
+    {
+        private readonly string _identityProvidersBaseUrl = "/api/identityProviders";
+        [Fact]
+        public async Task UsersController_Search_InvalidIdentityProvider_ReturnsBadRequest()
```

Comment:
Is the test method name accurate? It doesn't appear to match the actual test.

Reply:
Good catch - i'll fix that.